pkg/services/store/object/object.proto

```diff
@@ -7,17 +7,12 @@ message GRN {
   // the tenant/org id
   int64 tenant_id = 1;
 
-  // mabybe "namespace"?  valid values include
-  // * entity
-  // * drive
-  // * service/xyz
-  string scope = 2;
-
   // Identify the object kind.  This kind will be used to apply a schema to the body and
   // will trigger additional indexing behavior.
   string kind = 3;
 
   // Unique ID
+  // less then 40, no slashes or other special characters
```

Comment:
The UID constraints are still in discussion -- the existing UIDs (for dashboards) are limited to 40 characters and do not have slashes or special characters.

I think we will actually expand the rules so that the UIDs match k8s values:
https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names